src/features/vault.ts

```diff
@@ -0,0 +1,173 @@
+import * as vscode from 'vscode';
+import * as utilAnsibleCfg from './utils/ansibleCfg';
+import * as tmp from 'tmp';
+import * as cp from 'child_process';
+import * as fs from "fs";
+
+async function askForVaultId(ansibleCfg: string) {
+  let vaultId: string = "default";
+  if (ansibleCfg === "ANSIBLE_VAULT_IDENTITY_LIST") {
+    let envVar : any = process.env.ANSIBLE_VAULT_IDENTITY_LIST;
+    var identityList = envVar.split(',').map( (id : string) => id.split('@', 2)[0].trim());
+  } else {
+    let cfg : any = utilAnsibleCfg.getValueByCfg(ansibleCfg);
+    var identityList = cfg.defaults.vaultIdentity_list.split(',').map( (id : string) => id.split('@', 2)[0].trim());
```

Comment:
Why use `var` instead of `let`? In fact it looks like you might be able to use `const`.

Reply:
fixed in c1be20b